src/test/java/cosminsechel/tests/tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,93 @@
+        driver = BrowserManager.createDriver();
+        driver.get("https://demoqa.com/browser-windows");
+        System.out.println("Navigated to Demo QA window page");
+    }
+
+    public static void clickNewTabButton() {
+        String parentTab = driver.getWindowHandle();
+        WebElement newTabButton = driver.findElement(By.id("tabButton"));
+        newTabButton.click();
+        Set<String> tabs = driver.getWindowHandles();
+        for (String tab : tabs) {
+            if(!tab.equals(parentTab)) {
+                driver.switchTo().window(tab);
+                WebElement newTabHeading = driver.findElement(By.id("sampleHeading"));
+                System.out.println("Text on the new tab: " + newTabHeading.getText());
+                driver.close();
+            }
+        }
+        driver.switchTo().window(parentTab);
+    }
+
+    public static void interactWithElementsOnParentTab() {
+        WebElement clickElementButton = driver.findElement(By.id("click"));
+        clickElementButton.click();
+        WebElement generateAlertButton = driver.findElement(By.id("alert"));
+        generateAlertButton.click();
+        String alertText = driver.switchTo().alert().getText();
+        System.out.println("Alert text: " + alertText);
+        driver.switchTo().alert().accept();
+        WebElement switchToTabButton = driver.findElement(By.id("newTab"));
+        switchToTabButton.click();
+        String childTab = driver.getWindowHandle();
+        Set<String> tabs = driver.getWindowHandles();
+        for (String tab : tabs) {
+            if(!tab.equals(childTab)) {
+                driver.switchTo().window(tab);
+                WebElement childTabHeading = driver.findElement(By.id("childTab"));
+                System.out.println("Text on the child tab: " + childTabHeading.getText());
+                driver.close();
+            }
+        }
+        driver.switchTo().window(childTab);
+        WebElement mouseHoverButton = driver.findElement(By.id("mousehover"));
+        Actions action = new Actions(driver);
+        action.moveToElement(mouseHoverButton).perform();
+        WebElement mouseHoverText = driver.findElement(By.id("mousehovertext"));
+        System.out.println("Mouse hover text: " + mouseHoverText.getText());
+        WebElement draggableButton = driver.findElement(By.id("draggable"));
+        WebElement droppableButton = driver.findElement(By.id("droppable"));
+        action.dragAndDrop(draggableButton, droppableButton).build().perform();
+        WebElement droppedText = driver.findElement(By.id("dropped"));
+        System.out.println("Dropped text: " + droppedText.getText());
+        WebElement sliderButton = driver.findElement(By.id("slider"));
+        action.dragAndDropBy(sliderButton, 50, 0).build().perform();
+        WebElement sliderText = driver.findElement(By.id("sliderValue"));
+        System.out.println("Slider text: " + sliderText.getText());
+    }
+    public static void closeBrowser() {
```

Comment:
After you added the method from BrowserManager to close the driver delete this method from here. Thanks!